src/infection-model/index.js

```diff
@@ -0,0 +1,122 @@
+// this function comes from Justine, adapted from python model
+function simulateOneDay({
+  x,
+  N,
+  R0,
+  dIncubation = 2,
+  dInfectious = 4.1,
+  dRecoveryMild = 9.9,
+  dHospitalLag = 2.9,
+  dHospitalRecovery = 22,
+  dHospitalFatality = 8.3,
+  pSevereCase = 0.26,
+  pFatalityRate = 0.026,
+}) {
+  // Compute the updated SEIR values after 1 day
+  // x: [1D array of integers] counts for the 9 SEIR (susceptible/exposed/infectious/recovered) values
+  // N: [integer] population total per age bucket
+  // R0: [float] rate of spread factor
+  // D_incubation: [float] days from virus exposure to infectious/contagious state
+  // D_infectious: [float] days in infectious period
+  // D_recovery_mild: [float] days from end of infectious period to end of virus
+  // D_hospital_lag: [float] days from end of infectious period to hospital admission
+  // D_hospital_recovery: [integer] days from hospital admission to hospital release (non-fatality scenario)
+  // D_hospital_fatality: [float] days from hospital admission to deceased (fatality scenario)
+  // P_severe_case: [float] probability case will be severe enough for the hospital
+  // P_fatality_rate: [float] probability of mortality per age bucket
+
+  const alpha = 1 / dIncubation;
+  const beta = R0 / dInfectious;
+  const gamma = 1 / dInfectious;
+
+  const pMildCase = 1 - pSevereCase;
+
+  const susceptible = x[0];
+  const exposed = x[1];
+  const infectious = x[2];
+  const mild = x[3];
+  const severe = x[4];
+  const hospitalized = x[5];
+  const mildRecovered = x[6];
+  const severeRecovered = x[7];
+  const fatalities = x[8];
+
+  // Compute the deltas from the bottom of the tree up, additions to the leaf nodes are subtracted from their parent node as people flow from the root to the leaves of the tree
+  const mildRecoveredData = mild / dRecoveryMild;
+  const fatalitiesDelta =
+    ((pFatalityRate / pSevereCase) * hospitalized) / dHospitalFatality;
+  const severeRecoveredDelta =
+    ((1 - pFatalityRate / pSevereCase) * hospitalized) / dHospitalRecovery;
+
+  const mildDelta = gamma * pMildCase * infectious - mild / dRecoveryMild;
+  const severeDelta = gamma * pSevereCase * infectious - severe / dHospitalLag;
+  const hospitalizedDelta =
+    severe / dHospitalLag - fatalitiesDelta - severeRecoveredDelta;
+
+  const infectiousDelta = alpha * exposed - gamma * infectious;
+
+  // The transition from susceptible -> exposed depend on the total infectious population
+  const totalInfectious = infectious;
```

Comment:
This may seem weird here (why assign one var to another var with no modification?) but it's a placeholder for when this method evolves to incorporate the age distribution.

Reply:
Might be worth adding a TODO in that case, so it's obvious where the extension point is.